buildconfig/CMake/Packaging/osx/mantidpython.in

```diff
@@ -24,14 +25,25 @@ if [ -n "${PYTHONPATH}" ]; then
 	LOCAL_PYTHONPATH=${LOCAL_PYTHONPATH}:${PYTHONPATH}
 fi
 
+# Define MANTIDPATH
+MANTIDPATH="${INSTALLDIR}"
 if [ -n "$1" ] && [ "$1" = "--classic" ]; then
     shift
-    PROG=@PYTHON_EXECUTABLE@
+
+    set -- @WRAPPER_PREFIX@@PYTHON_EXECUTABLE@ $*@WRAPPER_POSTFIX@
+
+else
+    IPYTHON_STARTUP="import IPython;IPython.start_ipython()"
+
+    set -- @WRAPPER_PREFIX@@PYTHON_EXECUTABLE@ -c "${IPYTHON_STARTUP}" $*@WRAPPER_POSTFIX@
+
 fi
 
-# Define MANTIDPATH
-MANTIDPATH="${SCRIPT_PATH}"
 
-PYTHONPATH=${LOCAL_PYTHONPATH} \
-    MANTIDPATH=$MANTIDPATH \
-    ${PROG} $*
+LD_PRELOAD=${LOCAL_PRELOAD} \
```

Comment:
Can we remove `LD_PRELOAD`, `LD_LIBRARY_PATH` as these are not used on macOS.

`QT_API` should also no longer be required so can we remove that?